client/state/themes/actions/activate-theme.js

```diff
@@ -35,6 +36,7 @@ export function activateTheme(
 			.post( `/sites/${ siteId }/themes/mine`, {
 				theme: themeId,
 				...( dontChangeHomepage && { dont_change_homepage: true } ),
+				...( isEnabled( 'themes/template-persistent' ) && { persist_template: true } ),
```

Comment:
[nit] It might be better if the feature flag contains something like `theme-switch`

Reply:
Good catch